presto-main/src/main/java/com/facebook/presto/operator/scalar/VarbinaryFunctions.java

```diff
@@ -315,6 +315,38 @@ public static long crc32(@SqlType(StandardTypes.VARBINARY) Slice slice)
         return crc32.getValue();
     }
 
+    @Description("compute fnv-1 32 bit")
+    @ScalarFunction
+    @SqlType(StandardTypes.BIGINT)
+    public static long fnv1_32(@SqlType(StandardTypes.VARBINARY) Slice slice)
```

Comment:
ditto for this method name :)

Reply:
It seems for sql functions, we use _ a lot, and only using lower case. Should we keep the old method names like ScalarFunction("fnv1_32")?